pbpy/pbtools.py

```diff
@@ -140,94 +154,146 @@ def remove_file(file_path):
 
 
 def error_state(msg=None, fatal_error=False):
-    if msg != None:
+    if msg is not None:
         pblog.error(msg)
     if fatal_error:
-        # That was a fatal error, until issue is fixed, do not let user run PBSync
+        # This is a fatal error, so do not let user run PBSync until issue is fixed
         with open(error_file, 'w') as error_state_file:
             error_state_file.write("1")
-    out = input("Logs are saved in " +
-                pbconfig.get("log_file_path") + ". Press enter to quit...")
+    pblog.info(f"Logs are saved in {pbconfig.get('log_file_path')}. Press enter to quit...")
```

Comment:
Please also remove Press enter to quit... from the log

Reply:
Fixed